src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/DbIdVersionEntityId.java

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.ampt2d.metadata.persistence.entities;
+
+import javax.persistence.Embeddable;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.io.Serializable;
+import java.util.Objects;
+
+@Embeddable
+public class DbIdVersionEntityId implements Comparable<DbIdVersionEntityId>, Serializable {
+
+    @Size(min = 1, max = 32)
+    @NotNull
+    private String db;
+
+    @Size(min = 1, max = 32)
+    @NotNull
+    private String id;
+
+    @Size(min = 1, max = 32)
```

Comment:
display default version to be 1 in swagger

Reply:
Fixed.